EventFilter/GEMRawToDigi/interface/VFATdata.h

```diff
@@ -1,5 +1,6 @@
 #ifndef EventFilter_GEMRawToDigi_VFATdata_h
 #define EventFilter_GEMRawToDigi_VFATdata_h
+#include "stdint.h"
```

Comment:
system includes are typically in the brackets ```<stdint.h>```

Reply:
fixed here https://github.com/cms-sw/cmssw/pull/23189. I'd separate this pr into pieces